public/app/features/plugins/admin/components/PluginListItemBadges.tsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
 
+import { PluginType } from '@grafana/data';
 import { HorizontalGroup, PluginSignatureBadge } from '@grafana/ui';
 
 import { CatalogPlugin } from '../types';
 
-import { PluginEnterpriseBadge, PluginDisabledBadge, PluginInstalledBadge, PluginUpdateAvailableBadge } from './Badges';
+import {
+  PluginEnterpriseBadge,
+  PluginDisabledBadge,
+  PluginInstalledBadge,
+  PluginUpdateAvailableBadge,
+  PluginAngularBadge,
+} from './Badges';
 
 type PluginBadgeType = {
   plugin: CatalogPlugin;
 };
 
 export function PluginListItemBadges({ plugin }: PluginBadgeType) {
+  // Currently renderer plugins are not supported by the catalog due to complications related to installation / update / uninstall.
+  const hasUpdate = plugin.hasUpdate && !plugin.isCore && plugin.type !== PluginType.renderer;
```

Comment:
I really liked that you moved this from this component to the call site for better overview!